caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -181,6 +191,66 @@ func parseTLS(h Helper) ([]ConfigValue, error) {
 					cp.Curves = append(cp.Curves, h.Val())
 				}
 
+			case "clients":
+				ch := h.NewFromNextSegment() // new helper for client auth config
+				cp.ClientAuthentication = &caddytls.ClientAuthentication{}
+				for ch.Next() {
+					for ch.NextBlock(0) {
+						switch ch.Val() {
+						case "mode":
+							for ch.NextArg() {
+								cp.ClientAuthentication.Mode = ch.Val()
+							}
+						case "trusted_ca_certs":
+							for ch.NextArg() {
+								if _, err := caddytls.DecodeBase64DERCert(ch.Val()); err != nil {
+									return nil, h.Errf("cannot decode trusted CA certificate '%s'", ch.Val())
+								}
+								cp.ClientAuthentication.TrustedCACerts = append(cp.ClientAuthentication.TrustedCACerts, ch.Val())
+							}
+						case "trusted_ca_certs_file":
```

Comment:
This rename also flows better with the `_file` variants.
```suggestion
						case "trusted_ca_cert_file":
```